subsys/net/lib/nrf_cloud/Kconfig

```
@@ -158,6 +158,34 @@ config NRF_CLOUD_GATEWAY
 	  Enables functionality in this device to be compatible with
 	  nRF Cloud LTE gateway support.
 
+config NRF_CLOUD_DOWNLOADS
+	bool
+	default y
+	depends on NRF_CLOUD_FOTA || NRF_CLOUD_PGPS || NRF_CLOUD_FOTA_POLL
+	help
+	  Enable nRF Cloud file downloads for FOTA and P-GPS.
+
+choice NRF_CLOUD_DOWNLOADS_PROTOCOL
+	prompt "Select protocol to be used for nRF Cloud file downloads"
```

A choice by nature is a selection of one among many.
```suggestion
	prompt "nRF Cloud file download protocol"
```